pkg/validate/validate_test.go

```diff
@@ -49,6 +49,28 @@ func TestClusterName(t *testing.T) {
 	}
 }
 
+func TestVSphereClusterName(t *testing.T) {
+	cases := []struct {
+		name        string
+		clusterName string
+		valid       bool
+	}{
+		{"single lowercase", "a", true},
+		{"has a dot", "a.a", false},
+		{"valid name", "abcde", true},
```

Comment:
Would be good to add a test where cluster name has other characters like "ab:cd-ef". I am assuming this is supported.

Reply:
The main aim of the VsphereClusterName function is to just check the dot and then call ClusterName function. These cases should be tested in the TestClusterName function. I can add it there.